app/resources/media_relationship_resource.rb

```diff
@@ -1,5 +1,7 @@
 class MediaRelationshipResource < BaseResource
   attribute :role
+  
```

Comment:
`server` needs an [`.editorconfig`](https://github.com/hummingbird-me/hummingbird-client/blob/the-future/.editorconfig) like `client` @NuckChorris!